### ISSUE TITLE
[template/*] HTML template rule disables use JS comments instead of HTML comments

### ISSUE THREAD

**Issue by ChrisMBarr** [angular-eslint/angular-eslint#2547]
**Description and reproduction of the issue**
HTML Template rules insert a JS style comment to disable the rule, but they should be inserting an HTML style comment instead

![Image](https://github.com/user-attachments/assets/8224aa30-7e05-4e39-9514-3e1bc4029e1b)

**Actual Output**
```html
// eslint-disable-next-line @angular-eslint/template/label-has-associated-control
```

**Expected Output**
```html
<!-- eslint-disable-next-line @angular-eslint/template/label-has-associated-control -->
```

----

**Versions**

| package                                  | version |
| ---------------------------------------- | ------- |
| `@angular-eslint/eslint-plugin-template` | `20.1.1` |
| `@angular-eslint/template-parser`        | `20.1.1` |
| `@typescript-eslint/parser`              | `8.35.0` |
| `ESLint`                                 | `9.29.0` |
| `node`                                   | `22.14.0` |

```sh
Angular CLI: 20.0.3
Node: 22.14.0
Package Manager: npm 11.4.1
OS: win32 x64

Angular: 20.0.4
... animations, common, compiler, compiler-cli, core, forms
... localize, platform-browser, platform-browser-dynamic, router

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.2000.3
@angular-devkit/core         20.0.3
@angular-devkit/schematics   20.0.3
@angular/build               20.0.3
@angular/cli                 20.0.3
@schematics/angular          20.0.3
zone.js                      0.15.1
```
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by JamesHenry** [comment]
This needs to be reported on the VSCode extension, not here.

I can see it grabs the comment to use based on a language defaults mapping: https://github.com/microsoft/vscode-eslint/blob/c7c2ebf24c9e09313ab12a0792887d83b4e82827/server/src/eslintServer.ts#L496-L497

It looks like this has no way to support opening and closing comment markers for a line comment, and I can see the HTML one incorrectly has `//` in the mapping: https://github.com/microsoft/vscode-eslint/blob/c7c2ebf24c9e09313ab12a0792887d83b4e82827/server/src/languageDefaults.ts#L19

Another thing I am inferring here is that even if this gets fixed for HTML, it wouldn't work for inline templates because the document language would come through as TypeScript due to it being a .ts file.

Definitely worth some discussion with the maintainer once you open the issue

**Comment by ChrisMBarr** [comment]
Ah interesting, ok I was a little unsure of where this was coming from. The same will be true for CSS/SCSS as some ESLint plugins support those languages as well. Thanks.
